### ISSUE TITLE
Prefixed linear-gradient may get dropped if declaring multiple backgrounds

### ISSUE THREAD

**Issue by ngyikp** [jakubpawlowicz/clean-css#523]
Node version: v0.12.2
cleancss version: 3.1.9

Test script:

```javascript
var CleanCSS = require('clean-css');

var test = function(source) {
	console.log(new CleanCSS().minify(source).styles);
};

var t = '';

// correct:
t = 'p {' +
		'background-image: -webkit-linear-gradient(top, #fff, #000);' +
		'background-image: -moz-linear-gradient(top, #fff, #000);' +
		'background-image: -o-linear-gradient(top, #fff, #000);' +
		'background-image: linear-gradient(to bottom, #fff, #000);' +
	'}';
test(t);

// correct:
t = 'p {' +
		'background-image: -webkit-linear-gradient(top, #fff, #000), url(a.jpg);' +
		'background-image: -moz-linear-gradient(top, #fff, #000), url(a.jpg);' +
		'background-image: -o-linear-gradient(top, #fff, #000), url(a.jpg);' +
		'background-image: linear-gradient(to bottom, #fff, #000), url(a.jpg);' +
	'}';
test(t);

// fail:
t = 'p {' +
		'background-image: url(a.jpg), -webkit-linear-gradient(top, #fff, #000);' +
		'background-image: url(a.jpg), -moz-linear-gradient(top, #fff, #000);' +
		'background-image: url(a.jpg), -o-linear-gradient(top, #fff, #000);' +
		'background-image: url(a.jpg), linear-gradient(to bottom, #fff, #000);' +
	'}';
test(t);
```

Output:
```css
p{background-image:-webkit-linear-gradient(top,#fff,#000);background-image:-moz-linear-gradient(top,#fff,#000);background-image:-o-linear-gradient(top,#fff,#000);background-image:linear-gradient(to bottom,#fff,#000)}

p{background-image:-webkit-linear-gradient(top,#fff,#000),url(a.jpg);background-image:-moz-linear-gradient(top,#fff,#000),url(a.jpg);background-image:-o-linear-gradient(top,#fff,#000),url(a.jpg);background-image:linear-gradient(to bottom,#fff,#000),url(a.jpg)}

p{background-image:url(a.jpg),linear-gradient(to bottom,#fff,#000)}
```

**Comment by jakubpawlowicz** [comment]
Good catch, it's wrong indeed! :+1:

**Comment by jakubpawlowicz** [comment]
Fixed with #490 in master now.
